expo-zustand-styled-components/src/screens/Repository/Blob/Blob.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { View, Text } from 'react-native'
+
+const Blob = () => {
+  return (
+    <View>
+      <Text>Blob</Text>
+    </View>
+  )
+}
+
+export default Blob
```

Comment:
new line.

Maybe running `npm format` (if the command is configure on the project) helps